scripts/lua2dox.lua

```diff
@@ -361,6 +361,12 @@ local function process_line(line, in_stream, generics)
     return process_block_comment(line:sub(5), in_stream)
   end
 
+  -- Hax... I'm sorry
```

Comment:
Still needed now that it's not exposed?

Reply:
Yes still needed so `vim.query.get` and `vim.query.parse` are doc'd correctly.